src/erlcloud_as.erl

```diff
@@ -8,11 +8,13 @@
 %% AWS Autoscaling functions
 -export([describe_groups/0, describe_groups/1, describe_groups/2, describe_groups/4,
          set_desired_capacity/2, set_desired_capacity/3, set_desired_capacity/4,
-
-         describe_launch_configs/0, describe_launch_configs/1, describe_launch_configs/2, 
+         set_as_host/1,
```

Comment:
see erlcloud_autoscalling module too - seems half of it is already there.
i know it's strange we have that ddb vs ddb2 and _as vs _autoscalling but we just never got to deprecating things...

Reply:
secondly, i actually do encourage to switch to autoscalling to switch to SignV4 methods and dump SingV2. only SimpleDb will remain on SignV2, rest will die.